src/Uno.UI/UI/Xaml/Shapes/Path.wasm.cs

```diff
@@ -30,7 +33,63 @@ protected override SvgElement GetMainSvgElement()
 						("d", gd.Data),
 						("fill-rule", gd.FillRule == FillRule.EvenOdd ? "evenodd" : "nonzero"));
 					break;
+				case PathGeometry pg:
+					_path.SetAttribute(("d", ToStreamGeometry(pg)));
+					break;
+			}
+		}
+		/// <summary>
+		/// Transform the figures collection into a SVG Path according to :
+		/// https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d
+		/// </summary>
+		/// <param name="geometry"></param>
+		/// <returns></returns>
+		private string ToStreamGeometry(PathGeometry geometry)
+		{
+			CultureInfo ci = CultureInfo.InvariantCulture;
+			List<string> pathlist = new List<string>();
+
+			foreach (PathFigure figure in geometry.Figures)
+			{
+				pathlist.Add(string.Format(ci, "M " + figure.StartPoint.X + "," + figure.StartPoint.Y));
```

Comment:
Still does not work :)

```suggestion
				pathlist.Add(FormattableString.Invariant($"M{figure.StartPoint.X},{figure.StartPoint.Y}"));
```

You can use `using static System.FormattableString` at the top if you don't want to repeat `FormattableString`.